inc/saivlan.h

```diff
@@ -98,6 +98,12 @@ typedef enum _sai_vlan_attr_t
       * Value Range SAI_SWITCH_ATTR_VLAN_USER_META_DATA_RANGE */
     SAI_VLAN_ATTR_META_DATA,
 
+    /**
+      * The Multicast 802.1BR E-Channel Id (ECID), to be used in the flooding traffic
+      * sent by the CB to Port Extenders (PE). [sai_uint32_t] (CREATE_AND_SET) (default to 0)
+      */
+    SAI_VLAN_ATTR_FLOODING_ECID,
```

Comment:
@rsivasankar Now we're on the same page. : )   
"each Vlan must be associated with a list of {Cascading Port, ECID} tuple" --  this is exactly what I mean. My opinion is that we should allow the possibility that ControlStack allocates same ECID for two E-Channel in different Internal Bridge Port Extenders within one CB, to keep conformance to the IEEE standard. 
I remember that DELL has mentioned that only CB replication is required for now. In this case, this SAI_VLAN_ATTR_FLOODING_ECID attribute is not needed. Multicast ECID is only need when doing PE replication. I'm OK that DELL allocates different ECID for every ExtendedPort but I'm not OK with this SAI_VLAN_ATTR_FLOODING_ECID attribute.